### PR TITLE
fix: stay loading when refresh token  fail

### PR DESCRIPTION
**Pull request by QiShaoXuan**
fix: [#1193](https://github.com/toeverything/AFFiNE/issues/1193)

### CODE REVIEW

File: apps/web/src/components/message-center-handler/index.tsx [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 import { ReactNode, useCallback, useEffect } from 'react';
 
 import { useGlobalState } from '@/store/app';
 
+const clearAuth = (dataCenter: DataCenter, providerName: string) => {
+  const affineProvider = dataCenter.providers.find(p => p.id === providerName);
+  if (affineProvider && affineProvider instanceof AffineProvider) {
+    affineProvider.apis.auth.clear();
+  } else {
+    console.error('cannot find affine provider, please fix this ASAP');
```
Comment (pengx17) [inline apps/web/src/components/message-center-handler/index.tsx:14]: Please use `DebugLogger` in `'@affine/debug'`

**Review by darkskygit — APPROVED** [review]

File: apps/web/src/components/message-center-handler/index.tsx [hunk 2]
```diff
@@ -36,9 +37,16 @@ export function MessageCenterHandler({ children }: { children?: ReactNode }) {
           await router.push('/');
           router.reload();
         }
+
+        if (message.code === MessageCenter.messageCode.refreshTokenError) {
```
Comment (pengx17) [inline apps/web/src/components/message-center-handler/index.tsx:41]: It seems `refreshTokenError` and `noPermission` errors are handled in the same way right now (other than the error toast). Maybe we can merge the codes?

Comment (QiShaoXuan) [reply apps/web/src/components/message-center-handler/index.tsx:41]: RefreshTokenError is handled as a temporary and quick solution, with complex logic added later

**Review by pengx17 — APPROVED** [review]
I left several insignificant issues. Free free to ignore them if it is urgent.

**Pull request merged**